digitaltwin_client/samples/digitaltwin_sample_device_info/digitaltwin_sample_device_info.c

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-// Implements a sample DeviceInfo DigitalTwin interface.  As the name implies, this interface returns
-// information about the device such as the OS version, amount of storage, etc.
-//
+// Implements a sample DeviceInfo DigitalTwin interface of dtmi:azure:DeviceManagement:DeviceInformation;1.
+// As the name implies, this interface returns information about the device such as the OS version, amount of storage, etc.
```

Comment:
too wordy

```suggestion
// This interface returns information about the device such as the OS version, amount of storage, etc.
```

Reply:
As the code I shall soon checkin implies, I have fixed this.